CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/CollectionRequestHandler.swift

```diff
@@ -0,0 +1,142 @@
+//RequestHandler.swift
+//CBLTestServer-iOS
+
+import Foundation
+import CouchbaseLiteSwift
+
+public class CollectionRequestHandler {
+    public static let VOID: String? = nil
+    fileprivate var _pushPullReplListener:NSObjectProtocol?
+    
+    public func handleRequest(method: String, args: Args) throws -> Any? {
+        switch method {
+        case "collection_defaultCollection":
+            let database: Database = (args.get(name:"database"))!
+            let collection: Collection = try (database.defaultCollection())!
+            return collection
+            
+        case "collection_getCollectionName":
+            let collection: Collection = (args.get(name:"collection"))!
+            return collection.name
+            
+        case "collection_createCollection":
+            let scopeName: String = (args.get(name: "scopeName"))!
+            let collectionName: String = (args.get(name: "collectionName"))!
+            let database: Database = args.get(name: "database")!
+            let collection: Collection = try database.createCollection(name: collectionName, scope: scopeName)
+            return collection
+            
+        case "collection_collectionNames":
```

Comment:
curious, why only collection names, you could get all collection instance itself?

Reply:
this is made following CBL-C as well. So that same test case is executable on both C as well as IOS

Reply:
ok